src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+public class DeleteCommand extends Command {
+
+    private int num;
+
+    /**
+     * Constructor of the DeleteCommand class.
+     * @param num The index number of the Task in the TaskList that stores it.
+     */
+    public DeleteCommand(int num) {
+        this.num = num;
+    }
+
+    /**
+     * To delete the Task from the TaskList.
+     * @param taskList The TaskList that stores the Task.
+     * @param ui The Ui instance that will interact with the user.
+     * @param storage The Storage instance that will update the file.
+     */
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) {
```

Comment:
Perhaps expressing functionality through the method name might be more explanatory like maybe "executeTask" or "executeAction"?